admin/htop/Makefile

```
@@ -33,7 +33,7 @@ define Package/htop
   SECTION:=admin
   CATEGORY:=Administration
   TITLE:=Interactive processes viewer
-  DEPENDS:=+libncurses
+  DEPENDS:=+libncurses $(if $(CONFIG_HTOP_LMSENSORS),+libsensors)
```

Why so complicated? :-)

This one should work:
```suggestion
  DEPENDS:=+libncurses +HTOP_LMSENSORS:libsensors
```

Not tested, though.